src/main/java/core/basesyntax/service/file/FileWriterHandler.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.service.file;
+
+public interface FileWriterHandler {
```

Comment:
class name is overwhelmed
```suggestion
public interface FileWriter {
```